plugins/notifications/interactive.js

```diff
@@ -0,0 +1,92 @@
+const { notificationImage, icons } = require("./utils");
+const getSongControls = require('../../providers/song-controls');
+const notifier = require("node-notifier");
+
+//store song controls reference on launch
+let controls;
+let onPause;
+
+//Save controls and onPause option
+module.exports.setupInteractive = (win, unpauseNotification) => {
+    const { playPause, next, previous } = getSongControls(win);
+    controls = { playPause, next, previous };
+
+    onPause = unpauseNotification;
+
+    win.webContents.once("closed", () => {
+        Delete()
+    });
+}
+
+//delete old notification
+let toDelete;
+function Delete() {
```

Comment:
nit: functions are generally written lower-case, capital case being kept for classes
```suggestion
function deleteNotification() {
```

Reply:
missed this one, thanks :)